userguide/tasks.rst

```diff
@@ -184,6 +184,9 @@ path to the command is shown:
 When scheduling a script, test the script first to verify it is
 executable and achieves the desired results.
 
+.. note:: The default %brand% shell is :file:`zsh`, but
```

Comment:
I don't think we should even mention the default interactive shell here, just which one is used for scripts.  sh is a command, so uses different markup.  Please build-test this.

```suggestion
.. note:: Init/shutdown scripts are run with :command:`sh`.
```